source/site-administrators/studio/publishing-assets-in-external-storage.rst

```diff
@@ -0,0 +1,248 @@
+:is-up-to-date: True
+
+.. index:: Publishing Assets in External Storage
+
+.. _publishing-assets-in-external-storage:
+
+=====================================
+Publishing Assets in External Storage
+=====================================
+
+Crafter CMS supports managing assets in external storage through workflow and publishing mechanics.
+This allows uploading assets to an external storage for preview, that can then be published to either a live or a staging (depending on if staging is setup for your Crafter install) external storage, thus making the external assets available to delivery only after the assets have been published to the live external storage.
+
+The external storage could be in the cloud, such as AWS S3 or some other storage solution that is outside of where Crafter CMS is installed.
+
+--------------------------------
+Configuring the External Storage
+--------------------------------
+First we'll need to setup the external storage to be used by Crafter CMS.
+To setup an external storage for assets, open the **Sidebar**, then click on |siteConfig| -> *Configurations*.  Select ``Blob Stores`` from the dropdown and fill in the required information.
+
+.. code-block:: xml
+
+   <blobStore>
+     <id/>
+     <type/>
+     <pattern/>
+     <mappings>
+       <mapping>
+         <publishingTarget/>
+         <storeTarget/>
+         <prefix/>
+       </mapping>
+     </mappings>
+     <configuration/>
+   </blobStore>
+
+|
+
+To see more information on the Blob Stores configuration, see :ref:`blob-stores-configuration`
+
+After setting up the ``Blob Stores`` configuration, you may now use the external storage for uploading using the various upload methods provided by Crafter Studio, and publishing to live or staging if it's setup.
+
+-------
+Example
+-------
+
+Let's take a look at an example of setting up an external storage for preview, staging and live and then uploading and finally publishing assets to the external storage we setup.  In the example, we will use AWS S3 as the external storage and the Website Editorial blueprint in Crafter Studio to create our site.
+
+**Prerequisites:**
+
+#. Site created using the Website Editorial blueprint.
+#. Two AWS S3 buckets. One for authoring and another for delivery.  For our example, the following buckets were setup in AWS S3: *my-authoring-bucket* for authoring and *my-deli-bucket* for delivery.
```

Comment:
Actually it can be used with a single bucket, as long as preview, staging & live all have a unique prefix. Just in case you want to simplify the example or mention this.